.env.example

```
@@ -0,0 +1,14 @@
+CONTRACTS_BUILD_PATH=/app/build/ # Path to the build folder of the solidity smart contracts
+CONTRACTS_PATH=/app/contracts/ # Path to the solidity smart contracts
+FILES_TO_COMPILE="NrVerify,SchnorrSignature" # List of solidity smart contracts to compile
```

Mentioned in README.MD
```suggestion
FILES_TO_COMPILE="EllipticCurve,SchnorrSignature,SchnorrSignatureNodeRing"
```